### PR TITLE
Resize disk in `google_workbench_instance` resource

### PR DESCRIPTION
**Pull request by bcreddy-gcp**
fixes b/338962537

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
workbench: added update support to `gce_setup.boot_disk` and `gce_setup.data_disks` fields in `google_workbench_instance` resource
```

### CODE REVIEW

**Comment by bcreddy-gcp** [comment]
https://github.com/GoogleCloudPlatform/magic-modules/pull/10933

File: mmv1/templates/terraform/pre_update/workbench_instance.go.erb [hunk 1]
```diff
@@ -55,3 +55,35 @@ if stopInstance{
...
+}
+if d.HasChange("effective_labels") {
+    newUpdateMask = append(newUpdateMask, "labels")
+}
+
+// Overwrite the previously set mask.
+url, err = transport_tpg.AddQueryParams(url, map[string]string{"updateMask": strings.Join(newUpdateMask, ",")})
+if err != nil {
+    return err
+}
```
Comment (zli82016) [inline mmv1/templates/terraform/pre_update/workbench_instance.go.erb:81]: Duplicate code as in lines 1 - 35

**Comment by zli82016** [comment]
Is there a Github issue or Buganizer issue for this PR? If yes, can you please add it to the description section of this PR? Thanks.

**Comment by bcreddy-gcp** [comment]
b/338962537

File: mmv1/templates/terraform/constants/workbench_instance.go [hunk 2]
```diff
@@ -195,3 +195,48 @@ func waitForWorkbenchOperation(config *transport_tpg.Config, d *schema.ResourceD
 	return nil
 }
 <% end -%>
+
+func resizeWorkbenchInstanceDisk(config *transport_tpg.Config, d *schema.ResourceData, project string, userAgent string, isBoot bool) (error) {
```
Comment (zli82016) [inline mmv1/templates/terraform/constants/workbench_instance.go:199]: Do you mind putting the whole function inside `<% unless compiler == "terraformgoogleconversion-codegen" -%>` ?

**Comment by zli82016** [comment]
LGTM. One minor issue is that there are 28 commits for this PR, which could cause merging to main branch fail. Is it easy to merge all of the commits to one?

**Review by zli82016 — APPROVED** [review]
LGTM

**Pull request merged**